src/rod/sdf/joint.py

```diff
@@ -9,12 +9,15 @@
 
 @dataclasses.dataclass
 class Limit(Element):
-    lower: float = dataclasses.field(
-        metadata=mashumaro.field_options(serialize=Element.serialize_float)
+
+    lower: Optional[float] = dataclasses.field(
```

Comment:
Yep I didn't do it in this PR since I wanted to be consistent with the other fields. I'm not addressing this comment now, let's do this in bulk in another PR.